web-ui/src/main/java/com/tutrit/webclient/controller/CarController.java

```diff
@@ -14,8 +14,12 @@
 @RequestMapping("/cars")
 public class CarController {
 
-    @Autowired(required = false)
-    private CarGateway carGateway;
+    private final CarGateway carGateway;
```

Comment:
changes to `web-ui` module are out of the scope of this issue

Reply:
OK, I put it back